charts/merlin/README.md

```diff
@@ -99,6 +99,10 @@ Kubernetes-friendly ML model management, deployment, and serving.
 | imageBuilder.resources.requests.cpu | int | `1` |  |
 | imageBuilder.resources.requests.memory | string | `"512Mi"` |  |
 | imageBuilder.retention | string | `"48h"` |  |
+| imageBuilder.serviceAccount.annotations | object | `{}` |  |
+| imageBuilder.serviceAccount.create | bool | `true` |  |
```

Comment:
We don't have a template for creating a new SA. Is this intentional?

Reply:
What do you mean by template?

Reply:
I think, He meant that this chart is not creating the SA if the above value is true. It is only being used in the helper to create the name and in the merlin-deployment to add the serviceAcc name.

Reply:
ah, my bad, just added the template.